integrations/gitlab_v2/utils.py

```diff
@@ -0,0 +1,39 @@
+from collections.abc import AsyncGenerator
+from typing import Any, Callable
+
+from loguru import logger
+
+
+def extract_merge_request_payload(data: dict[str, Any]) -> dict[str, Any]:
+    logger.info(f"Extracting merge request for project: {data['project']['id']}")
+    return {
+        "id": data["object_attributes"]["id"],
+        "title": data["object_attributes"]["title"],
+        "author": {
+            "name": data["user"]["name"],
+        },
+        "state": data["object_attributes"]["state"],
+        "created_at": data["object_attributes"]["created_at"],
+        "updated_at": data["object_attributes"]["updated_at"],
+        "web_url": data["object_attributes"]["source"]["web_url"],
+        "reviewers": data["reviewers"][0]["name"],
+        "__project": data["project"],
+    }
+
+
+def extract_issue_payload(data: dict[str, Any]) -> dict[str, Any]:
+    logger.info(f"Extracting issue for project: {data['project']['id']}")
+    return {
+        "id": data["object_attributes"]["id"],
+        "title": data["object_attributes"]["title"],
+        "link": data["object_attributes"]["url"],
+        "description": data["object_attributes"]["description"],
+        "created_at": data["object_attributes"]["created_at"],
+        "updated_at": data["object_attributes"]["updated_at"],
+        "author": {
+            "name": data["user"]["name"],
+        },
+        "state": data["object_attributes"]["state"],
```

Comment:
You are not expected to perform data mapping manually, take advantage of the `ocean.register_raw(kind, [resource_data])` to map the data